doc/classes/Camera2D.xml

```diff
@@ -55,6 +41,20 @@
 				Returns the camera limit for the specified [enum Side]. See also [member limit_bottom], [member limit_top], [member limit_left], and [member limit_right].
 			</description>
 		</method>
+		<method name="get_screen_center" qualifiers="const">
+			<return type="Vector2" />
+			<description>
+				Returns the center of screen for this [Camera2D], in global coordinates.
+				[b]Note:[/b] The exact targetting position of the camera may be different. See [method get_target_position].
+			</description>
+		</method>
+		<method name="get_target_position" qualifiers="const">
+			<return type="Vector2" />
+			<description>
+				Returns the location of the [Camera2D]'s tracked point, in global coordinates.
```

Comment:
Reading this line I first thought that this was the point the camera is looking at (in 3D), which obviously doesn't make any sense in 2D.
Maybe mention that this is used for targeting during the smoothing?

Reply:
This is the position the camera is looking at, or at least trying to look at. So renaming to `get_target_position()` definitely makes sense.

Note: In #63773, I've suggested the following description:
> Returns the camera's target position relative to the origin.
 [b]Note:[/b] The returned value is not the same as [member Node2D.global_position], as it is affected by the drag properties. It is also not the same as the current position if [member smoothing_enabled] is [code]true[/code] (see [method get_camera_screen_center]).

Reply:
Changed the PR to use a description very similar to @madmiraal 's